dashboard/templates/assignments.html

```diff
@@ -7,14 +7,18 @@
 
 {% block content %}
 {% if course_view_option.show_assignment_planning %}
+<div ng-controller="mainController" ng-app="visApp">
 <div class="container-fluid">
     <h5 style="padding-left: 10px">Progress toward Final Grade</h5>
     <div id="chart"></div>
+    <div ng-show="progress.length == 0" class="alert alert-info">
+        There's no data
```

Comment:
since we changed "data" into "plan", should the message in UI also say "There is no assignment plan", or "There is no assignment data for planning", or something similar?

Reply:
I changed the working for clarity